cw_decred/lib/wallet_addresses.dart

```diff
@@ -1,23 +1,43 @@
 import 'dart:convert';
+import 'package:mobx/mobx.dart';
 
 import 'package:cw_core/address_info.dart';
 import 'package:cw_core/wallet_addresses.dart';
 import 'package:cw_core/wallet_info.dart';
 import 'package:cw_decred/api/libdcrwallet.dart' as libdcrwallet;
 
-class DecredWalletAddresses extends WalletAddresses {
-  DecredWalletAddresses(WalletInfo walletInfo) : super(walletInfo);
+part 'wallet_addresses.g.dart';
 
-  String currentAddr = '';
```

Comment:
What if we left this variable and then in `performBackgroundTasks` We check for a new address, and if it's different than what we had saved here, we update the selectedAddr and this one? That way we can keep the default new.